mtg_card_identifier/CardScanner/CardScannerView.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  CardScannerView.swift
+//  mtg_card_identifier
+//
+//  Created by mac on 8/16/23.
+//
+
+import SwiftUI
+import VisionKit
+import AVFoundation
```

Comment:
It would be nice to restrict these import statements into the one file somehow. Maybe there is a binding that gets updated on .didAppear or maybe there is a delegate function you can tie into to determine if the value changes.